sile.in

```diff
@@ -53,7 +53,11 @@ if SILE.masterFilename then
     ProFi = require 'ProFi'
     ProFi:start()
   end
-  p,e = pcall(function() SILE.readFile(unparsed[1]) end)
+  if SILE.printTraceback then
```

Comment:
This bit would become something like:

```lua
p,e = xpcall(function() SILE.readFile(unparsed[1]) end, SILE.printTraceback)
```

i.e. Always using `xpcall()`. Does this have any disadvantages? It looks like the output of `pcall()` by default is pretty much the same as `xpcall(func, print)`. Am I missing some downside here?

Reply:
I am not sure, is it? I was under the impression that `pcall` did not print out anything and that the error message was returned as a second argument. Similarly, `debug.traceback` only returns the traceback, it does not print it.

So the equivalent would be something which returns the current error (and I don't know what that is). But as I said, I am not really sure how this whole thing works.